r/README.md

```diff
@@ -1,331 +1,104 @@
 
-Window functions (e.g. `ntile()`) are not yet
-supported. Inside `dplyr` verbs, Arrow offers support for many functions and
-operators, with common functions mapped to their base R and tidyverse
-equivalents. The [changelog](https://arrow.apache.org/docs/r/news/index.html)
-lists many of them. If there are additional functions you would like to see
-implemented, please file an issue as described in the [Getting
-help](#getting-help) section below.
+- Development versions of `arrow` are released nightly. Most users will not 
+need to install nightly builds, but if you do please see the article on [installing nightly builds]([installation guide](https://arrow.apache.org/docs/r/articles/install_nightly.html) for more information.
 
-For `dplyr` queries on `Table` objects, if the `arrow` package detects
-an unimplemented function within a `dplyr` verb, it automatically calls
-`collect()` to return the data as an R `data.frame` before processing
-that `dplyr` verb. For queries on `Dataset` objects (which can be larger
-than memory), it raises an error if the function is unimplemented;
-you need to explicitly tell it to `collect()`.
+## Arrow resources 
 
-### Additional features
-
-Other applications of `arrow` are described in the following vignettes:
-
--   `vignette("python", package = "arrow")`: use `arrow` and
-    `reticulate` to pass data between R and Python
--   `vignette("flight", package = "arrow")`: connect to Arrow Flight RPC
-    servers to send and receive data
--   `vignette("arrow", package = "arrow")`: access and manipulate Arrow
-    objects through low-level bindings to the C++ library
-
-The Arrow for R [cheatsheet](https://github.com/apache/arrow/blob/-/r/cheatsheet/arrow-cheatsheet.pdf) and [Cookbook](https://arrow.apache.org/cookbook/r/index.html) are additional resources for getting started with `arrow`.
+In addition to the official [Arrow R package documentation](https://arrow.apache.org/docs/r/), the [Arrow for R cheatsheet](https://github.com/apache/arrow/blob/-/r/cheatsheet/arrow-cheatsheet.pdf), and the [Apache Arrow R Cookbook](https://arrow.apache.org/cookbook/r/index.html) are useful resources for getting started with `arrow`.
 
 ## Getting help
 
 If you encounter a bug, please file an issue with a minimal reproducible
 example on the [Apache Jira issue
 tracker](https://issues.apache.org/jira/projects/ARROW/issues). Create
-an account or log in, then click **Create** to file an issue. Select the
-project **Apache Arrow (ARROW)**, select the component **R**, and begin
-the issue summary with **`[R]`** followed by a space. For more
-information, see the **Report bugs and propose features** section of the
+an account or log in, then click "Create" to file an issue. Select the
+project "Apache Arrow (ARROW)", select the component "R", and begin
+the issue summary with "[R]" followed by a space. For more
+information, see the "Report bugs and propose features" section of the
 [Contributing to Apache
```

Comment:
This'll soon be github, but we can change it when we get that up and running

Reply:
I am so looking forward to this: not gonna lie jira gives me headaches